crates/aiken-project/src/tests/gen_uplc.rs

```diff
@@ -2487,101 +2487,205 @@ fn acceptance_test_27() {
 }
 
 #[test]
-fn acceptance_test_30() {
+fn acceptance_test_28_unique_empty_list() {
     let src = r#"
-      pub fn foldr(xs: List<a>, f: fn(a, b) -> b, zero: b) -> b {
+      pub fn filter(xs: List<a>, f: fn(a) -> Bool) -> List<a> {
         when xs is {
           [] ->
-            zero
+            []
           [x, ..rest] ->
-            f(x, foldr(rest, f, zero))
+            if f(x) {
+              [x, ..filter(rest, f)]
+            } else {
+              filter(rest, f)
+            }
         }
       }
       
-      pub fn concat(left: List<a>, right: List<a>) -> List<a> {
-        foldr(left, fn(x, xs) { [x, ..xs] }, right)
-      }
-      
-      pub fn flat_map(xs: List<a>, f: fn(a) -> List<b>) -> List<b> {
+      pub fn unique(xs: List<a>) -> List<a> {
         when xs is {
           [] ->
             []
           [x, ..rest] ->
-            concat(f(x), flat_map(rest, f))
+            [x, ..unique(filter(rest, fn(y) { y != x }))]
         }
       }
       
-      test flat_map_2() {
-        flat_map([1, 2, 3], fn(a) { [a, a] }) == [1, 1, 2, 2, 3, 3]
-      }
+      test unique_1() {
+        unique([]) == []
+      }  
     "#;
```

Comment:
I am not sure to understand the rationale here? Why are we replacing existing tests by some others?

Reply:
I had a duplicate test with the wrong name. I replaced the duplicate.